crates/core/tedge_agent/src/operation_workflows/actor.rs

```diff
@@ -253,9 +253,14 @@ impl WorkflowActor {
 
                 // Move to the next state to await the builtin operation outcome
                 let new_state = state.update(handlers.on_exec.clone());
+                info!("Publishing new state: {} to MQTT", &new_state.status);
```

Comment:
This log and the next one are remnants of some debugging. To be removed before merge.